documentation/source/building.rst

```diff
@@ -302,16 +302,24 @@ The following variables are makefile variables, not environment variables.
 
 .. make:var:: VHDL_SOURCES_<lib>
 
-      A list of the VHDL source files to include in the VHDL library *lib* (currently for GHDL/ModelSim/Questa/Xcelium/Incisive only).
+      A list of the VHDL source files to include in the VHDL library *lib* (currently for GHDL/ModelSim/NVC/Questa/Xcelium/Incisive only).
 
 .. make:var:: VHDL_LIB_ORDER
 
-      A space-separated list defining the order in which VHDL libraries should be compiled (needed for ModelSim/Questa/Xcelium/Incisive, GHDL determines the order automatically).
+      A space-separated list defining the order in which VHDL libraries should be compiled (needed for ModelSim/NVC/Questa/Xcelium/Incisive, GHDL determines the order automatically).
 
 .. make:var:: COMPILE_ARGS
 
       Any arguments or flags to pass to the compile stage of the simulation.
 
+.. make:var:: ANALYZE_ARGS
+
+      Any arguments or flags to pass to the analysis stage of the simulation (NVC only).
+
+.. make:var:: ELAB_ARGS
+
+      Any arguments or flags to pass to the elaboration stage of the simulation (NVC only).
+
```

Comment:
I'm not against these arguments, but I wonder what we're doing for other simulators, and how ANALYZE_ARGS/ELAB_ARGS is related to COMPILE_ARGS.

I know we had some discussions on GHDL about the right granularity of build steps. I'm neither a GHDL user nor do I remember the discussion exactly, but maybe @themperek or @marlonjames might do?

Reply:
In many ways Questa also has those semantics. There is compilation, optimization (includes elaboration), then simulation. Perhaps what needs to happen here is ANALYZE_ARGS becomes COMPILE_ARGS, then we need to add ELAB_ARGS and use those in the other simulators.

Reply:
2 sounds like a good plan to me. We can do compile and opt in one step with different sets of args.

Reply:
To summarize, for this PR the request is to remove ANALYZE_ARGS and use the existing COMPILE_ARGS instead. ELAB_ARGS remains as it is proposed in this PR and is planned to be used by other simulators in the future (not part of this PR, though).